faq/tidb-faq.md

```diff
@@ -248,7 +248,9 @@ Check the time difference between the machine time of the monitor and the time w
 
 #### Deploy TiDB offline using TiDB Ansible
 
-It is not recommended to deploy TiDB offline using TiDB Ansible. If the control machine has no access to external network, you can deploy TiDB offline using TiDB Ansible. For details, see [Offline Deployment Using TiDB Ansible](/offline-deployment-using-ansible.md).
+> **Warning:**
+>
+> TiDB Ansible deployment method is deprecated since TiDB v4.0. [Use TiUP to deploy TiDB](/production-offline-deployment-using-tiup.md) instead.
```

Comment:
```suggestion
> It is not recommended to deploy TiDB using TiDB Ansible since TiDB v4.0. [Use TiUP to deploy TiDB](/production-offline-deployment-using-tiup.md) instead.
```